src/dps-email-poller/src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 server.servlet.context-path=/dpsemailpoller
+server.port=8989
```

Comment:
do we need to worry about this in openshift?

Reply:
Thanks for catching that.